image_classification/README.en.md

```diff
@@ -395,106 +313,131 @@ The following are the components of `resnet_cifar10`:
+The learning rate adjustment policy can be defined with variables `learning_rate_decay_a`($a$), `learning_rate_decay_b`($b$) and `learning_rate_schedule`. In this example, discrete exponential method is used for adjusting learning rate. The formula is as follows,
+$$  lr = lr_{0} * a^ {\lfloor \frac{n}{ b}\rfloor} $$
+where $n$ is the number of processed samples, $lr_{0}$ is the learning_rate.
 
-<p align="center">
-<img src="image/plot_en.png" width="400" ><br/>
-Figure 12. The error rate of VGG model on CIFAR10
-</p>
+### Training
 
-## Model Application
+`cifar.train10()` will yield records during each pass, after shuffling, a batch input is generated for training.
 
-After training is done, the model from each pass is saved in `output/pass-%05d`. For example, the model of Pass 300 is saved in `output/pass-00299`. The script `classify.py` can be used to extract features and to classify an image. The default config file of this script is `models/vgg.py`.
+```python
+reader=paddle.reader.batch(
+    paddle.reader.shuffle(
+        paddle.dataset.cifar.train10(), buf_size=50000),
+        batch_size=128)
+```
 
+`feeding` is devoted to specify the correspondence between each yield record and `paddle.layer.data`. For instance,
+ the first column of data generated by `cifar.train10()` corresponds to image layer's feature.
 
-### Prediction
+```python
+feeding={'image': 0,
+         'label': 1}
+```
 
-We can run the following script to predict the category of an image. The default device is GPU. If to use CPU, set `-c`.
+Callback function `event_handler` is used to track training and testing process that might be triggered once the action to which it is attached is executed.
 
-```bash
-python classify.py --job=predict --model=output/pass-00299 --data=image/dog.png # -c
+```python
+# event handler to track training and testing process
+def event_handler(event):
+    if isinstance(event, paddle.event.EndIteration):
+        if event.batch_id % 100 == 0:
+            print "\nPass %d, Batch %d, Cost %f, %s" % (
+                event.pass_id, event.batch_id, event.cost, event.metrics)
+        else:
+            sys.stdout.write('.')
+            sys.stdout.flush()
+    if isinstance(event, paddle.event.EndPass):
+        result = trainer.test(
+            reader=paddle.reader.batch(
+                paddle.dataset.cifar.test10(), batch_size=128),
+            reader_dict={'image': 0,
+                         'label': 1})
+        print "\nTest with Pass %d, %s" % (event.pass_id, result.metrics)
 ```
 
-Here is the result：
+Finally, we can invoke `trainer.train` to start training:
 
-```text
-Label of image/dog.png is: 5
+```python
+trainer.train(
+    reader=reader,
+    num_passes=200,
+    event_handler=event_handler,
+    feeding=feeding)
 ```
 
-### Feature Extraction
+Here is an example log after training for one pass. The average error rates are 0.6875 on training set and 0.8852 on validation set.
```

Comment:
on training set -> on the training set
on validation set -> on the validation set